_sources/SequenceMutation/Mutability.rst

```diff
@@ -0,0 +1,184 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: seqmut-1-
+   :start: 1
+
+.. index:: mutable, immutable
+   runtime error; string item assignment
+   string; immutable
+
+Mutatability
+============
+
+Some Python collection types - strings and lists so far - are able to mutate and some are not. If 
+a type is able to mutate, then it is said to be mutable. If the type is not able to mutate then it
+is said to be immutable. This will be expanded below.
+
+Strings are Immutable
+---------------------
+
+One final thing that makes strings different from some other Python collection types is that
+you are not allowed to modify the individual characters in the collection. It is tempting to use 
+the ``[]`` operator on the left side of an assignment, with the intention of changing a character 
+in a string. For example, in the following code, we would like to change the first letter of 
+``greeting``.
+
+.. activecode:: ac8_1_1
+    
+    greeting = "Hello, world!"
+    greeting[0] = 'J'            # ERROR!
+    print(greeting)
+
+Instead of producing the output ``Jello, world!``, this code produces the
+runtime error ``TypeError: 'str' object does not support item assignment``.
+
+Strings are **immutable**, which means you cannot change an existing string. The
+best you can do is create a new string that is a variation on the original.
+
+.. activecode:: ac8_1_2
+    
+    greeting = "Hello, world!"
+    newGreeting = 'J' + greeting[1:]
+    print(newGreeting)
+    print(greeting)          # same as it was
+
+The solution here is to concatenate a new first letter onto a slice of
+``greeting``. This operation has no effect on the original string.
```

Comment:
I think we want a sentence and another example even on re-assignment to show that if you do not want to be creating new variables all the time this is a perfectly acceptable way to do it.